src/main/java/com/github/tomakehurst/wiremock/http/Request.java

```diff
@@ -60,6 +60,14 @@ interface Part {
 
   QueryParameter queryParameter(String key);
 
+  default FormParameter formParameter(String key) {
```

Comment:
Can we ditch these defaults and ensure we've implemented these everywhere instead?

Reply:
Done. We have implemented these everywhere. Oleg suggested adding this as it might break the implementation for those who might be using this interface as this is part of the library and exposed.

Reply:
I feel that it is very unlikely that anyone using it as we are using this interface as base for our receiving requests, matching, and stubbing it wherever required. Consumers of Wiremock may not be using it for any of these purposes. But it is always better to add a safety net.